src/theme.rs

```diff
@@ -21,17 +21,16 @@ impl From<ThemeDescription> for Value {
         } = value;
         Value::record(
             record! {
-                "id" => Value::string(id, Span::unknown()),
+                "id" => Value::test_string(id),
                 "name" => match name {
-                    Some(name) => Value::string(name, Span::unknown()),
-                    None => Value::nothing(Span::unknown()),
-
+                    Some(name) => Value::test_string(name),
+                    None => Value::test_nothing(),
                 },
                 "author" => match author {
-                    Some(author) => Value::string(author, Span::unknown()),
-                    None => Value::nothing(Span::unknown()),
+                    Some(author) => Value::test_string(author),
+                    None => Value::test_nothing(),
                 },
-                "default" => Value::bool(default, Span::unknown()),
+                "default" => Value::test_bool(default),
```

Comment:
According to the docs, these `Value::test_*` should only be used for tests. Is the naming then just a bit odd if they are just values with unknown span?

Reply:
`Value::test_blah()` is just a shorthand way of providing a `Span::unknown()` automatically. It's identical under the hood. So, if you have no span, that's what you should do.

Reply:
Then it's a weird name in my opinion.

Reply:
It was originally intended for tests, but it still works for this purpose.